src/routes/swagger.rs

```diff
@@ -0,0 +1,14 @@
+use axum::Router;
+use utoipa::OpenApi;
+use utoipa_swagger_ui::SwaggerUi;
+
+use super::health;
+use super::teapot;
+
+pub fn route() -> Router {
+    Router::new().merge(SwaggerUi::new("/swagger-ui").url("/openapi.json", ApiDoc::openapi()))
```

Comment:
Why the `-ui` suffix in the path? Do you plan on exposing an otherwise non-ui endpoint, or is this some convention i'm unaware of?

Reply:
From what I've inferred when reading the docs, this is the standard path used to expose Swagger UI (i.e. an interactive API documentation compliant with the Swagger standard). I won't insist on keeping it that way, but I'd rather follow general conventions, unless we have a good reason not to.